test/integration/backward_compatible/sql/ExecuteTest.js

```diff
@@ -499,6 +499,7 @@ describe('SqlExecuteTest', function () {
             client = await Client.newHazelcastClient({
                 clusterName: cluster.id
             });
+            TestUtil.markClientVersionAtLeast(this, '4.2.1');
```

Comment:
We haven't released the `4.2.1` client yet, and maybe we never will :smile: 

I think we could make this `5.0.0` and update it if we release the 4.2.1

Reply:
Also, can you put the comment you put below to this as well?